rustfmt.toml

```diff
@@ -1 +1,2 @@
 tab_spaces=2
+edition = "2018"
```

Comment:
Not sure if there is another way to allow `rust-fmt` deal with 2018 (read: `async` code), but this shouldn't hurt anything, right?

Reply:
Oh ok, that makes sense. I wonder what I might have done to my machine that made it complain for me...